<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileBox</class>
 <widget class="QWidget" name="ProfileBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">-webkit-box-shadow: -10px 0px 13px -7px #000000, 10px 0px 13px -7px #000000, -15px 4px 8px -2px rgba(0,0,0,0.12); 
box-shadow: -10px 0px 13px -7px #000000, 10px 0px 13px -7px #000000, -15px 4px 8px -2px rgba(0,0,0,0.12);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
border: 4px ridge #462e23;

};

</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>21</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="miniPhoto">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>7</y>
        <width>85</width>
        <height>85</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>85</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #462e23;
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>85</width>
        <height>85</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Courier New&quot;, Courier, monospace;
font-size: 20px;
letter-spacing: 6px;
word-spacing: 0px;
color: #000000;
font-weight: normal;
text-decoration: none;
font-style: normal;
font-variant: normal;
text-transform: none;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Name">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Name_2">
         <property name="text">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Name_3">
         <property name="text">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>miniPhoto</zorder>
     <zorder>pushButton</zorder>
     <zorder>widget_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
